backend-lib/src/main/rust/lib.rs

```diff
@@ -157,17 +157,22 @@ fn account_id_from_jni<'local, P: Parameters>(
 pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_initOnLoad<'local>(
     _env: JNIEnv<'local>,
     _: JClass<'local>,
-    show_trace_logs: jboolean
+    trace_logs_level: jint
 ) {
     // Set up the Android tracing layer.
     #[cfg(target_os = "android")]
     let android_layer = paranoid_android::layer("cash.z.rust.logs")
         .with_ansi(false)
-        .with_filter(if show_trace_logs == JNI_TRUE {
-            tracing_subscriber::filter::LevelFilter::TRACE
-        } else {
-            tracing_subscriber::filter::LevelFilter::DEBUG
-        });
+        .with_filter(
+            if trace_logs_level == 1 {
+                tracing_subscriber::filter::LevelFilter::TRACE
+            } else if trace_logs_level == 0 {
+                tracing_subscriber::filter::LevelFilter::DEBUG
+            } else {
+                tracing_subscriber::filter::LevelFilter::OFF
+            }
```

Comment:
The `tracing-subscriber` levels have corresponding numeric levels that we should match if we're going to use numbers. But the code to parse them only takes strings, and at that point we should just use those strings across the FFI, i.e. `"off"`, "debug"` etc., and then have a helper that uses `env.get_string(log_level).to_str()?.parse()?` to get a `LevelFilter`.

Reply:
Sounds good. I updated the Kotlin side and the Rust side function interface + documentation comment. Could you please update the function body as described above? Maybe we could also leverage the `from_str` API? https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.LevelFilter.html#method.from_str

Reply:
That API is what the `.parse()` is using.